product/carbon-home/conf/identity/profile-mapping.yaml

```diff
@@ -123,4 +123,30 @@ profiles:
    readonly: false
    dataType: "email"
    regex: "*"
-   defaultValue: "user@example.com"
\ No newline at end of file
+   defaultValue: "user@example.com"
+ -
+  profileName: "username-recovery"
+  isAdminProfile: true
+  claims:
+  -
+   claimURI: "http://wso2.org/claims/givenname"
+   displayName: "First Name"
+   required: true
+   readonly: false
+   regex: "*"
+   dataType: "text"
+  -
+   claimURI: "http://wso2.org/claims/lastname"
+   displayName: "Last Name"
+   required: true
+   readonly: false
+   regex: "*"
+   dataType: "text"
+  -
+   claimURI: "http://wso2.org/claims/email"
+   displayName: "Email"
+   required: true
+   readonly: false
+   regex: "*"
```

Comment:
Use a suitable regex for email address.

Reply:
This is used in the all config, this is only working. Need to refactor all in the next implementation